### PR TITLE
Added n_root benchmark and doc

### PR DESCRIPTION
**Pull request by Andrey-Yao**

### CODE REVIEW

**Comment by sampsyo** [comment]
Looks like a good start! But I think the profiling output is missing. Can you make sure you added everything, and try running the tests locally?

**Comment by Andrey-Yao** [comment]
I have run the program through brili and it worked. The reason I didn't include the .prof file was that Turnt wasn't working for me (maybe I was using it wrong?) Turnt is installed, but when I run `turnt --save n_root.bril`  or `turnt --save n_root.json` it gives error "KeyError: 'command'"

**Comment by sampsyo** [comment]
Yes, you will want to get Turnt working (otherwise, you'll be playing guess-and-check with the CI on GitHub).

Can you include full details? It is impossible to see what's going on without the full traceback.

**Comment by Andrey-Yao** [comment]
Of course. Here you go:

[andrey@SpectreSeahorse lib]$ turnt --save n_root.bril
1..1
Traceback (most recent call last):
  File "/home/andrey/.local/bin/turnt", line 8, in <module>
    sys.exit(turnt())
  File "/home/andrey/.local/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/andrey/.local/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/andrey/.local/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/andrey/.local/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/andrey/.local/lib/python3.10/site-packages/turnt.py", line 342, in turnt
    sc, msg = run_test(path, config, idx + 1, save, diff, verbose, dump, args)
  File "/home/andrey/.local/lib/python3.10/site-packages/turnt.py", line 257, in run_test
    cmd, out_files, return_code = load_options(config, path, args)
  File "/home/andrey/.local/lib/python3.10/site-packages/turnt.py", line 182, in load_options
    get_command(config, path, contents, args),
  File "/home/andrey/.local/lib/python3.10/site-packages/turnt.py", line 62, in get_command
    cmd = extract_single_option(contents, 'cmd') or config['command']
KeyError: 'command'

**Comment by sampsyo** [comment]
Looks like your test file is in a directory called ``lib`. I'm not sure where that is, but you'll want to run the `turnt` command on a file that's inside the Bril tests directory, so it's associated with the `turnt.toml` configuration file there.

**Comment by Andrey-Yao** [comment]
Ooops. Rookie mistake. Turnt is working now. I just pushed the missing file.

File: benchmarks/n_root.out [hunk 1]
```diff
@@ -0,0 +1 @@
+1.5157178699512095
```
Comment (sampsyo) [inline benchmarks/n_root.out:1]: I'm guessing that the tests are failing because of the missing newline here? Trying `turnt --save` could help.

**Review by sampsyo — CHANGES_REQUESTED** [review]

**Pull request merged**